packages/elements-core/src/components/Layout/ResponsiveSidebarLayout.tsx

```diff
@@ -80,15 +80,17 @@ export const ResponsiveSidebarLayout = React.forwardRef<HTMLDivElement, Responsi
               style={{ width: '1em', flexBasis: '6px', cursor: 'ew-resize' }}
             />
           </Flex>
-        ) : (
-          <MobileTopNav onTocClick={onTocClick!} name={name} logo={logo} tree={tree!} pathname={pathname} />
         )}
 
         <Box ref={scrollRef} bg="canvas" px={!isResponsiveLayoutEnabled ? 24 : 6} flex={1} w="full" overflowY="auto">
           <Box style={{ maxWidth: `${maxContentWidth - currentSidebarWidth}px` }} py={16}>
             {children}
           </Box>
         </Box>
+
+        {isResponsiveLayoutEnabled && (
+          <MobileTopNav onTocClick={onTocClick!} name={name} logo={logo} tree={tree!} pathname={pathname} />
+        )}
```

Comment:
I moved the nav bar to after the docs, so it renders on top even without z-index hacking.